pages/common/aws-google-auth.md

```diff
@@ -0,0 +1,20 @@
+# aws-google-auth
+
+> A command-line tool allows you to acquire AWS temporary (STS) credentials using Google Apps as a federated (Single Sign-On, or SSO) provider.
+> More information: <https://github.com/cevoaustralia/aws-google-auth>.
+
+- Login with Google SSO:
+
+`aws-google-auth -u example@example.com -I $GOOGLE_IDP_ID -S $GOOGLE_SP_ID -d 3600`
+
+- Ask, which role to use (in case you have several SAML roles):
```

Comment:
```suggestion
- Login [a]sking which role to use (in case of several available SAML roles):
```
Little bit clearer.

Reply:
I haven't seen the `[a]sking` syntax in other pages? Seems fine to me though.

Reply:
I've seen it often enough:

	$ grep -r '\w\+\?\[.\]\w\+' pages

	pages/linux/xsel.md:- Use a command's output as input of the clip[b]oard (equivalent to `Ctrl + C`):
	pages/linux/pwgen.md:- Generate random password with s[y]mbols:
	pages/linux/isoinfo.md:- E[x]tract a specific file from an ISO image and send it out `stdout`:
	pages/common/perl.md:- Loo[p] over all lines of a file, editing them [i]n-place using a find/replace [e]xpression:
	pages/common/chmod.md:- Give the [u]ser who owns a file the right to e[x]ecute it:

Reply:
Ah, cool. No need to worry then :P